charts/apisix-ingress-controller/templates/deployment.yaml

```diff
@@ -46,6 +46,10 @@ spec:
             items:
               - key: config.yaml
                 path: config.yaml
+      initContainers:
+        - name: wait-apisix-admin
+          image: busybox:1.28
+          command: ['sh', '-c', "until nc -z {{ .Values.admin.serviceName }}.{{ .Release.Namespace }}.svc.cluster.local {{ .Values.admin.servicePort }} ; do echo waiting for apisix-admin; sleep 2; done;"]
```

Comment:
We'd better let users to specify the whole apisix admin service name instead of assuming the service is deployed inside Kubernetes and the cluster domain is `cluster.local`.